src/Lean/Parser/Command.lean

```diff
@@ -269,7 +269,7 @@ corresponding `end <id>` or the end of the file.
   "namespace " >> checkColGt >> ident
 /--
 `end` closes a `section` or `namespace` scope. If the scope is named `<id>`, it has to be closed
-with `end <id>`.
+with `end <id>`. Otherwise, the `end` command is optional.
```

Comment:
"Otherwise" makes it seem like "in contrast to the preceding" on first reading, so I suggest changing it.
```suggestion
with `end <id>`. The `end` command is optional at the end of a file.
```

Reply:
That's better, indeed! Changed.